.github/workflows/artillery-action-users-api.yml

```diff
@@ -0,0 +1,70 @@
+name: Artillery action to test function users
+
+# When there is a push to the development branch, this
+# GitHub action workflow will run
+on: 
+  push:
+    branches:
+      - development
+
+# Sets up the environment variables
+env:
+  NODE_VERSION: '14' # version of Node we are using
+  NPM_CONFIG_PREFIX: '~/.npm-global' # required to set path for Artillery call
+
+jobs:
+  # Sets up ubuntu VM for actions
+  setup:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Checks out current repo
+    - name: 'Checkout Repository GitHub Action'
+      uses: actions/checkout@v2
+
+    # Sets up Node
+    - name: Setup Node ${{ env.NODE_VERSION }} Environment
+      uses: actions/setup-node@v2
+      with:
+        node-version: ${{ env.NODE_VERSION }}
+        check-latest: true
+    
+    # Sets up NPM
+    - name: 'Run npm init'
+      run: npm init -y
+
+    # Uses AWS-Actions to set up AWS credentials
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-west-2
+
+    # Installs Artillery globally
+    - name: Run npm install artillery
+      run: npm install -g artillery
+
+    # Grabs the current date and time for naming convention
+    - name: Get current date
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d-%H-%M-%S')"
+
+    # # Takes a small break to make sure everything has been processed
+    # - name: Sleep
+    #   run: sleep 10s
+
+    # Call artillery test located at ./test/artillery/artillery-perf-test-users-user_id-tasks.yaml
+    # Return temp results to test/results/artillery-test-users-user_id-tasks-results.json
+    - name: Run Artillery Test
+      run: /home/runner/.npm-global/lib/node_modules/artillery/bin/artillery run --output test/results/artillery.json ./test/artillery/artillery-test-users-api.yml
```

Comment:
I saw some other previous work from classmates doing the /home/runner/.npm-global/[...] thing but I found it unnecessary in the end.

Reply:
I see, so would you just do `artillery run`?

Reply:
Whatever the artillery run command is, maybe. I was able to just `slsart invoke [...]` instead of all the preamble